### PR TITLE
chore(tree): remove flex style

### LINKED ISSUE #39194: Style of antd5 tree blockNode
### Reproduction link

[https://stackblitz.com/edit/react-vgqnqg?file=demo.tsx](https://stackblitz.com/edit/react-vgqnqg?file=demo.tsx)

### Steps to reproduce

set [blockNode] for tree

### What is expected?
the `.ant-tree-title` element width is not stretched
image

### What is actually happening?
the `.ant-tree-title` element width is stretched
image

| Environment | Info |
| --- | --- |
| antd | 5.0.3 |
| React | 18.2.0 |
| System | mac |
| Browser | Chrome |

---

### PR DESCRIPTION
**Pull request by Wxh16144**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

fix: #39194

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Fix Tree node width issue when it is `blockNode`. |
| 🇨🇳 Chinese | 修复 Tree `blockNode` 的节点占位宽度问题。   |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tree/style/index.tsx [hunk 1]
```diff
@@ -269,7 +269,6 @@ export const genBaseStyle = (prefixCls: string, token: TreeToken): CSSObject =>
         ${treeCls}-node-content-wrapper,
         ${treeCls}-checkbox + span
       `]: {
-        display: 'flex',
         flexWrap: 'nowrap',
```
Comment (afc163) [inline components/tree/style/index.tsx:272]: flexWrap 还有意义么

File: components/tree/style/index.tsx [hunk 2]
```diff
@@ -269,8 +269,6 @@ export const genBaseStyle = (prefixCls: string, token: TreeToken): CSSObject =>
         ${treeCls}-node-content-wrapper,
         ${treeCls}-checkbox + span
       `]: {
-        display: 'flex',
-        flexWrap: 'nowrap',
```
Comment (afc163) [inline components/tree/style/index.tsx:273]: 最好搞清楚当初为啥要加

**Review by afc163 — APPROVED** [review]

**Comment by zombieJ** [comment]
https://github.com/ant-design/ant-design/pull/35389/files#diff-78ef3b69572a75bd42d45f802abd7b3107409d07231819d2b576291f01a696fbR260
看起来不能直接删，wrapper 的 flex 还是要的

**Comment by Wxh16144** [comment]
> https://github.com/ant-design/ant-design/pull/35389/files#diff-78ef3b69572a75bd42d45f802abd7b3107409d07231819d2b576291f01a696fbR260 看起来不能直接删，wrapper 的 flex 还是要的

大致瞟了一眼，样式上没啥问题，是哪里会影响到么

**Pull request merged**